core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -366,4 +367,122 @@ class PlaintextConsumerTest extends BaseConsumerTest {
 
     consumer0.close()
   }
+
+  @Test
+  def testRoundRobinAssignment() {
+    // 1 consumer using round-robin assignment
+    this.consumerConfig.setProperty(ConsumerConfig.GROUP_ID_CONFIG, "roundrobin-group")
+    this.consumerConfig.setProperty(ConsumerConfig.PARTITION_ASSIGNMENT_STRATEGY_CONFIG, classOf[RoundRobinAssignor].getName)
+    val consumer0 = new KafkaConsumer(this.consumerConfig, new ByteArrayDeserializer(), new ByteArrayDeserializer())
+
+    def createTopicAndSendRecords(topicName: String, numPartitions: Int, recordsPerPartition: Int): Unit = {
```

Comment:
Can we move the other ```createTopicAndSendRecords``` out of the test case, let let these two functions to share it? For this use case we can just ignore the returned partition set.